### PR TITLE
chore(test): update protractor to 2.1

### PR DESCRIPTION
**Pull request by juliemr**

### CODE REVIEW

File: npm-shrinkwrap.clean.json [hunk 1]
```diff
@@ -207,7 +207,7 @@
           "dependencies": {
             "traceur": {
               "version": "0.0.33",
-              "resolved": "git+https://github.com/vojtajina/traceur-compiler#d90b1e34c799bf61cd1aafdc33db0a554fa9e617",
+              "resolved": "git://github.com/vojtajina/traceur-compiler.git#d90b1e34c799bf61cd1aafdc33db0a554fa9e617",
```
Comment (juliemr) [inline npm-shrinkwrap.clean.json]: To create this diff, I did `npm shrinkwrap --dev` and then `./scripts/npm/clean-shrinkwrap`. I'm not sure why this change happened. Does anyone know if I should just revert this line?

Comment (petebacondarwin) [inline npm-shrinkwrap.clean.json]: I wouldn't worry about it. It is still pointing to the same commit

Comment (petebacondarwin) [inline npm-shrinkwrap.clean.json]: Probably just an npm version change thingy. I would merge this change

**Comment by juliemr** [comment]
Thanks! Merged as 3545abfc310b79964c86fe37411f41e61400626c
